src/conn/conn_handle.c

```diff
@@ -131,6 +131,9 @@ __wt_connection_destroy(WT_CONNECTION_IMPL *conn)
     __wt_free(session, conn->dhhash);
     __wt_free(session, conn->fhhash);
 
+    /* Free allocated checkpoint snapshot */
```

Comment:
How about `/* Free allocated recovered checkpoint snapshot */`?

Reply:
Updated the comment.